.github/workflows/lint-md.yml

```diff
@@ -8,4 +8,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: npm install
+      # https://github.com/Flet/github-slugger/pull/35
+      - run:
+          curl https://github.com/Flet/github-slugger/compare/master...jablko:patch-1.patch |
```

Comment:
I don't understand what this is doing. Is this a fork of github-slugger? Why use that instead of the normal one?

Reply:
https://github.com/Flet/github-slugger/pull/35 is waiting to be merged. This cherry picks that patch (applies it to the normal github-slugger) in the meantime.

Reply:
https://github.com/Flet/github-slugger is the normal one, `jablko:patch-1` is https://github.com/Flet/github-slugger/pull/35

Reply:
Let's wait and just add a dependency once Flet/github-slugger#35 has shipped. Do you want to merge this PR with just the markdown fixes?

Reply:
@sandersn Okay, I've reverted this bit and yes please, I'd like to merge this PR (now just the Markdown fixes).